tests/e2e/v2store_deprecation_test.go

```diff
@@ -58,6 +58,34 @@ func TestV2DeprecationNotYet(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestV2DeprecationWriteOnlyWAL(t *testing.T) {
```

Comment:
Note: this case won't work anymore if we bootstrap etcd based on v3store. But that isn't finalized yet, and may not be done in 3.6.  But please let's add a comment/note.

Reply:
We might need to update the test, however the test is still relevant.